pkg/sql/sem/builtins/fixed_oids.go

```diff
@@ -0,0 +1,2041 @@
+	`varbit_send(varbit: varbit) -> bytes`:                                                                                                                                          1938,
+	`varcharin(input: anyelement) -> varchar`:                                                                                                                                       1889,
+	`varcharout(varchar: varchar) -> bytes`:                                                                                                                                         1888,
+	`varcharrecv(input: anyelement) -> varchar`:                                                                                                                                     1887,
+	`varcharsend(varchar: varchar) -> bytes`:                                                                                                                                        1886,
+	`variance(arg1: int) -> decimal`:                                                                                                                                                230,
+	`variance(arg1: decimal) -> decimal`:                                                                                                                                            231,
+	`variance(arg1: float) -> float`:                                                                                                                                                232,
+	`version() -> string`:                                                                                                                                                           1284,
+	`voidin(input: anyelement) -> void`:                                                                                                                                             1997,
+	`voidout(void: void) -> bytes`:                                                                                                                                                  1996,
+	`voidrecv(input: anyelement) -> void`:                                                                                                                                           1995,
+	`voidsend(void: void) -> bytes`:                                                                                                                                                 1994,
+	`width_bucket(operand: decimal, b1: decimal, b2: decimal, count: int) -> int`:                                                                                                   811,
+	`width_bucket(operand: int, b1: int, b2: int, count: int) -> int`:                                                                                                               812,
+	`width_bucket(operand: anyelement, thresholds: anyelement[]) -> int`:                                                             
```

Comment:
nit: just return `existing max id + 1`, what do you think?
I guess the effort here is trying get any unused id in between, and that'd be helpful if we removed any builtin functions. I'd prefer `existing max id + 1` because: (1) removing builtin function could be super rare. (2) reuse an id seems not desirable since we're trying to make OIDs deterministic here.

Reply:
Good point, done.